themes/upbound/assets/scss/_code-theme.scss

```diff
@@ -2,13 +2,6 @@
 // https://gist.github.com/AbeEstrada/e16e4d22d9adec4e720fd62a16a0113d
 
 @mixin code-theme {
-
-
-  body{
```

Comment:
Note: It felt out of place to have our body style rule inside the code-theme mixin